### PR TITLE
Apply black to the Scripts/ directory

### PR DESCRIPTION
**Pull request by peterjc**
This pull request addresses issue #2008.

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

@svalqui would you like to review this?

### CODE REVIEW

**Comment by peterjc** [comment]
Grr. I need to switch back to Python 2 locally to work out what I broke under flake8...

**Comment by peterjc** [comment]
My attempts to modernise this line failed, flake8 under Python 2 was not happy (unsure why, perhaps a plugin fails):

https://github.com/biopython/biopython/blob/biopython-175/Scripts/Restriction/ranacompiler.py#L440

File: Scripts/debug/debug_blast_parser.py [hunk 1]
```diff
@@ -41,7 +42,9 @@
 
 -o    <testfile> is a BLAST output file.
 
-""" % sys.argv[0]
+"""
```
Comment (svalqui) [inline Scripts/debug/debug_blast_parser.py]: This closing triple quote doesn't align with the opening one. not sure if we want them aligned as for instance the parentheses do not align either, black style does that; I have mostly aligned DocStrings but assignments I might have left them like this.

File: Scripts/xbbtools/nextorf.py [hunk 2]
```diff
@@ -171,17 +178,18 @@ def GetCDS(self, seq, strand=1):
                 elif codon_type == STOP:
                     if start_site == 0:
                         continue
-#                    if codon == 'XXX': print('do something')
+                    #                    if codon == 'XXX': print('do something')
                     stop = pos + 2
-#                    print("stop")
+                    #                    print("stop")
                     length = stop - start_site + 1
```
Comment (svalqui) [inline Scripts/xbbtools/nextorf.py]: I think the comments should be closer to the hash as "# comment..."

File: Scripts/debug/debug_blast_parser.py (hunk 1 above)
Comment (peterjc) [reply Scripts/debug/debug_blast_parser.py]: I haven't explored the rules of the syntax to properly understand how Python interprets the indentation vs which leading spaces are part of the string values.  However, this does look like a style judgement - in which case do whatever black does is the easy solution.  This is worth discussing for #2008 as a whole.

File: Scripts/xbbtools/nextorf.py (hunk 2 above)
Comment (peterjc) [reply Scripts/xbbtools/nextorf.py]: This does look odd - and yes, worth a manual fix.

I think black assumed the lines would be: indentation, hash, space, code - but here they were hash, indentation, code.

**Comment by peterjc** [comment]
Thank you @svalqui for looking at this 👍

**Pull request merged**